sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/CliSuite.scala

```diff
@@ -125,4 +133,23 @@ class CliSuite extends FunSuite with BeforeAndAfterAll with Logging {
   test("Single command with -e") {
     runCliWithin(1.minute, Seq("-e", "SHOW DATABASES;"))("" -> "OK")
   }
+
+  test("Single command with --database") {
+    runCliWithin(1.minute)(
```

Comment:
@liancheng I have to pass multiple tuples like (a, b), (c, d),... for the queriesAndExpectedAnswers argument.
Thus, it should be used ( ... ), not { ... }.